sdk/formrecognizer/Azure.AI.FormRecognizer/tests/FormRecognizerClient/RecognizeBusinessCardsLiveTests.cs

```diff
@@ -344,7 +344,7 @@ public void StartRecognizeBusinessCardsThrowsForDamagedFile()
         [RecordedTest]
         public void StartRecognizeBusinessCardsFromUriThrowsForNonExistingContent()
         {
-            var client = CreateFormRecognizerClient();
+            var client = CreateFormRecognizerClient(retryFlakyErrors: false);
```

Comment:
In this test, we are reproducing one of the errors our policy retries. For this reason, I created the `retryFlakyErrors` parameter to tell the method whether we want to retry error or not.